<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menuWidget</class>
 <widget class="QWidget" name="menuWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>519</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QLabel#labelMenuDescription{
     border-style: outset;
     border-color: beige;
     border-width: 1px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4" rowspan="2">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>487</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>URW Gothic L</family>
       <pointsize>25</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string>color: rgb(234, 225, 228);</string>
     </property>
     <property name="text">
      <string>Choose a Menu Style</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>322</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>URW Gothic L</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string>color: rgb(234, 225, 228);</string>
     </property>
     <property name="text">
      <string>You can also customize your &lt;b&gt;KDE menu&lt;/b&gt; as you like. Please choose one from the following styles.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="pictureMenuStyles">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>320</width>
       <height>190</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>320</width>
       <height>190</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>URW Gothic L</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string>#pictureMenuStyles{
border: 1px solid rgb(255, 255, 255);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLabel" name="labelMenuDescription">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>122</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 81), stop:0.00537634 rgba(95, 34, 52, 80), stop:1 rgba(255, 213, 230, 80));
border: 1px solid rgb(236, 236, 236, 80);
padding:5px;
color: rgb(234, 225, 228);</string>
     </property>
     <property name="text">
      <string>Sample</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="3">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>834</width>
       <height>144</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QComboBox" name="menuStyles">
     <property name="font">
      <font>
       <family>URW Gothic L</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(95, 34, 52);</string>
     </property>
     <item>
      <property name="text">
       <string>Kick-off Menu</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Simple Menu</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lancelot Menu</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="2">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>321</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
